README.md

```diff
@@ -78,6 +78,21 @@ $ lineapy ipython
 
 This will automatically load the LineaPy extension in the corresponding interactive shell application.
 
+Or, if the application is already running without the extension loaded, you can load it

```

Comment:
if the application is already running without the extension loaded, which can happen when we start the Jupyter server with `jupyter notebook` or `jupyter lab` without `lineapy`,